{{ cookiecutter.project_slug }}/tests/test_package.py

```diff
@@ -15,4 +16,4 @@ def test_version_type():
 
 def test_version_number_is_semver():
     """Should be a semver."""
-    assert all([int(x) for x in __version__.split(".")])
+    assert re.match(r"\d+\.\d+\.\d+-{0,1}\w*", __version__) is not None
```

Comment:
`re.fullmatch(r"\d+\.\d+\.\d+(-\w+)?", __version__)` ist passender, sonst matchen auch `2.3.4-` oder `2.3.4xyz`

Reply:
Mh, `poetry` benutzt suffixes àla `1.1.0a0` für [prereleases](https://python-poetry.org/docs/cli/#version). Aber ja: `2.3.4-`darf nicht sein.

Reply:
Ok. aber es reicht ja aus, wenn wir das für uns festlegen.

p.s.: no idea, why I wrote this comment in german. 🤷

Reply:
https://semver.org/#spec-item-9 The dash seems to be required but there are some other corner cases

Reply:
OK, I think that's was quite enough of thinking for these subtleties :+1: 

> re.fullmatch(r"\d+\.\d+\.\d+(-\w+)?", __version__) ist passender, sonst matchen auch 2.3.4- oder 2.3.4xyz

That one it is, incluided in https://github.com/Leibniz-HBI/smormlpy/pull/20/commits/2c3d7134352e734ae08a2209826cfb3ce799bd31.